content/docs/try-react.md

```diff
@@ -0,0 +1,33 @@
+---
+id: try-react
+title: Try React
+permalink: docs/try-react.html
+next: add-react-to-a-new-app.html
+redirect_from:
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+Try React online or set up your local dev environment
+
+## Online
+
+If you're just interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s).
+
+## HTML Template
+
+If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
+
+## Install React
+
+- **[Add React to a New App](/docs/add-react-to-a-new-app.html)** Create a new app with a starter kit like [Create React App].(http://github.com/facebookincubator/create-react-app)
+- **[Add React to an Existing App](/docs/add-react-to-a-new-app.html)** _Advanced configuration._ Use React with a build system like Webpack.
```

Comment:
I don't think this description is very helpful.

Reply:
I'm not sure the dichotomy is correct here - "Add React to A New App" links to CRA primarily, and "Existing App" is all about build systems and CDN links. It makes sense to promote CRA as the starting point, but I think it is confusing to talk about the next page as only for existing apps. It's more of an advanced guide, or maybe a bundling/build guide. The CDN links could also probably be more prominent, with the caveat that they aren't very useful without Babel.

What do you think?

Reply:
I understand what you're saying. My point is just that 

>_Advanced configuration._ Use React with a build system like Webpack.

is not very meaningful to someone visiting the page for the first time. What advanced configuration? What is webpack? Does the first link not use webpack? etc I'm open to other phrasings of the same idea, this sentence just read as confusing to me.

>I think it is confusing to talk about the next page as only for existing apps

In a way, yes. But on the other hand it provides a clear entry point for each use case. If you're ready to configure things on your own you already know enough that you don't need either of these sections.

Reply:
Just changed the description to "Add React to a build system or larger app." --- how does that sound? Can iterate on the naming of the page later.

Also added a "Next Steps" header above the Quick Start and Install sections to make it clear that those aren't part of Trying React, but ideas for what to do next if you're set up.